tldw_Server_API/tests/Sharing/test_sharing_endpoints.py

```
@@ -0,0 +1,300 @@
+def test_app(test_user):
+    app = FastAPI()
+    app.include_router(router, prefix="/api/v1")
+
+    async def _fake_user():
+        return test_user
+
+    from tldw_Server_API.app.core.AuthNZ.User_DB_Handling import get_request_user
+    app.dependency_overrides[get_request_user] = _fake_user
+
+    # Disable RBAC rate limiting
+    def _no_rate_limit(scope: str):
```

Ruff (0.15.6)

[warning] Missing return type annotation for private function `_no_rate_limit`

(ANN202)